packages/medusa/src/services/order-edit.ts

```diff
@@ -315,6 +322,130 @@ export default class OrderEditService extends TransactionBaseService {
     })
   }
 
+  /**
+   * Create or update order edit item change line item and apply the quatity
+   * - If the item change already exists then update the quantity of the line item as well as the line adjustments and tax lines
+   * - If the item change does not exist then create the item change and cloned the original line item with the quantity as well as the line adjustments and tax lines
+   * @param orderEditId
+   * @param originalLineItemId
+   * @param data
+   */
+  async updateLineItem(
+    orderEditId: string,
+    originalLineItemId: string,
```

Comment:
todo: now that we have a fresh set of items we don't need to use the item id from the order. We can just use the id of the cloned item instead

Reply:
Indeed, now the item id sent to the api can be one of the items from the order edit 👌